### DISCUSSION TITLE
Migration Issue

### DISCUSSION THREAD

**Discussion by harveysm** [mealie-recipes/mealie#3101] [General]
Hello, I'm trying to migrate from frontend-v1.0.0beta-5 to V1. I have V1 up & running Ok, but I can't seem to migrate my backup created from beta-5. I have beta5 shutdown temporarily. 

I have tried uploading the backup file as is (this includes database.json, data folder, mealie folder) and I get a failure error. I have also tried rezipping just the mealie folder with recipes folder inside of it as it suggests on the migration page. But that also gives a failure. When I click on the failure, it just shows a report ID & nothing else. I also don't see anything in the logs about the failure. 

Am I doing something wrong, or can anyone provide some insight? Thanks! 

Running this on Synology, with Postgres16.

![image](https://github.com/mealie-recipes/mealie/assets/5342181/3a30deec-e705-40d1-85cf-48135b82bab1)

**Comment by Kuchenpirat** [comment]
Hi, you actually should be able to upgrade without having to destroy your instance by simply changing the version tag on the image to `v1.1.1`. Always make a backup when upgrading (see below how)

But if you want to upgrade and change your instance:
1. create a backup at `admin/backups`
2. Download that backup
3. Navigate to `admin/backups` on new instance
4. upload backup 
5. Restore

Your page seems to be the migration page which allows users from before v1 to migrate to the v1 version (v1.0.0beta-5 is considered v1 already)

**Reply by harveysm** [reply]
Oh geez, I did not realize that the migration was not the same as a backup. Your instructions worked flawlessly & I have everything in the new instance now. Thank you so much for all your work!

**Reply by Kuchenpirat** [reply]
hey, verry nice to hear :)

Have fun with mealie!
